### PR TITLE
mod: change package ioutil->os

### PR DESCRIPTION
**Pull request by dlstjq7685**
little change fs_mock.go.
`ioutil` is Deprecated, so i changed to `os` package.

### CODE REVIEW

**Comment by gen2thomas** [comment]
Hi @dlstjq7685 , thanks for preparation of the PR. Currently we are fighting with the download problem of gobot framework since v2+, so please be patient. No action is required, also no rebase at the moment.

After that phase I will have a look to your change. Meanwhile I can just say: there are some more usages for ioutil in the code and it would be nice if we can fix this all at once.

Thank you for your patience!

**Comment by gen2thomas** [comment]
Hi @dlstjq7685 , dev branch is now open for merge again. Please could you also fix the other findings for ioutil.

Thanks Thomas

**Comment by dlstjq7685** [comment]
@gen2thomas 
waiting for your comment.
please check this changes.

File: cli/generate.go
Comment (gen2thomas) [inline cli/generate.go:1]: this file was removed now, please rebase your branch (or simply revert the change on this file)

**Comment by gen2thomas** [comment]
Hi @dlstjq7685 , many thanks for your good work, please see my only comment regarding the conflict warning below.

**Comment by dlstjq7685** [comment]
@gen2thomas 
I was forgot merge current dev branch.
Please re-check commit files.

**Review by gen2thomas — APPROVED** [review]
looks good

**Comment by gen2thomas** [comment]
Nice work, thanks! Already merged.
